certified-connectors/OpenText eDOCS by One Fox/readme.md

```diff
@@ -0,0 +1,37 @@
+## OpenText eDOCS by One Fox
+Empower OpenText eDOCS as the primary repository for all things content related. Together with the OpenText eDOCS Power Automate Connector by One Fox you can streamline content processes across a widespread application landscape and limit information silos.
```

Comment:
Can "Power Automate Connector" be changed to "Power Platform Connector"? Thanks!

Reply:
@MatthijsOneFox Checking in on the above